actions/traits/LtiModuleTrait.php

```diff
@@ -39,9 +39,6 @@
      */
     protected function returnLtiError(\taoLti_models_classes_LtiException $error, $returnLink = true)
     {
-        // full trace of the error
-        \common_Logger::e($error->__toString());
```

Comment:
This is not debug message. We need it according to the story TAO-4032. In case if lti error occurred we need to show to the end user only ID of error, and this id will be used to easily find message in the log.